metabase/metabase
# add bundle size action

Closes https://github.com/metabase/metabase/issues/[issue_number]

### Description

This action will print a comment with bundle diff size and reject a PR with a huge diff, but to test it properly we have to merge it to master

https://github.com/andresz1/size-limit-action

### How to verify

- merge to master
- create a new PR where you add some new dependency (e.g. lodash)
- verify that comment from action appeared on the PR

### Demo

not at the moment
